pyproject.toml

```diff
@@ -25,9 +25,6 @@ dependencies = [
   "numpy!=2.0.0",  # https://github.com/lmcinnes/umap/issues/1138
   "pandas>=1.0",
   "jinja2",
-  "umap-learn",
```

Comment:
Lots of notebooks need updating. Maybe we add the extra then remove from core deps?

Reply:
reverted